crates/ruff_python_parser/src/string.rs

```diff
@@ -654,14 +663,14 @@ pub(crate) fn parse_strings(
                     deduped.push(value);
                 }
                 Expr::Constant(ast::ExprConstant {
-                    value: Constant::Str(inner),
+                    value: Constant::Str(StringConstant { value, .. }),
                     ..
                 }) => {
                     if current.is_empty() {
                         current_start = value_range.start();
                     }
                     current_end = value_range.end();
-                    current.push(inner);
+                    current.push(value);
```

Comment:
I believe we need to set the implicit concat flag in this case. But not entirely sure. But you can see how it expands the range of the current element.

Reply:
I think it's fine as-is, because `current` is just `Vec<String>`, and these get converted to `Expr::Constant` when pushed onto `deduped` (which sets the implicit flag).